src/Router.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import WinDetail from './Pages/Detail/WinDetail';
+import ImgUpload from './Pages/User/ImgUpload';
+
 import App from './App';
 
 function Router() {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<App />} />
+        <Route path="/win/winid" element={<WinDetail />} />
```

Comment:
일단 이 부분은 지워져야 할 것 같고 나중에 쓰신다면

```suggestion
        <Route path="/win/:winid" element={<WinDetail />} />
```

이렇게 써야할 것 같습니다.